src/client/WindshaftWorker.js

```diff
@@ -0,0 +1,29 @@
+import { MVTWorker } from '../sources/MVTWorker';
+import schema from '../renderer/schema';
+
+export class WindshaftWorker extends MVTWorker {
+    decodeProperty (metadata, propertyName, propertyValue) {
+        const basename = schema.column.getBase(propertyName);
+        const column = metadata.properties[basename];
+        if (!column) {
+            return;
+        }
+        switch (column.type) {
+            case 'date':
+            {
+                const d = new Date();
```

Comment:
Maybe these six lines could be packed in a little self-explaining function... and could also use destructuring with `const {min, max} = column`;

Reply:
:+1: